src/components/Tags/Tags.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div v-if="tags.length > 0">
+      <Tag v-for="tag in tags"
+           :key="tag"
+           :icon="icon"
+           :disabled="disabled"
+           :outlined="outlined"
+           :hovered="hovered"
+           :elevated="elevated"
+           :flat="flat"
+           :color="color"
+           class="mr-1"
+      >{{ tag }}</Tag>
+    </div>
+    <div v-else class="tag d-flex mr-1" >
+      <Tag
+          :icon="icon"
+          :disabled="disabled"
+          :outlined="outlined"
+          :elevated="elevated"
+          :hovered="hovered"
+          :color="color"
+          :flat="flat"
+          class="mr-1"
+      >--</Tag>
+    </div>
```

Comment:
I think we can update this to not need the v-if/v-else by making a computed property for the tags like this

```
internalTags() {
  return this.tags.length > 0 ? this.tags : ['--']
}
```

That way we can just v-for `internalTags` and won't need to duplicate all the props in the v-else.